app/graphql/mutations/add_user.rb

```diff
@@ -7,12 +7,10 @@ class AddUser < Mutations::BaseMutation
     def resolve(params:)
       user_params = Hash params
       begin 
-        user = User.create!(user_params)
-        { user: user }
-      rescue ActiveRecord::RecordInvalid => e 
-        GraphQL::ExecutionError.new("Invalid attributes for #{e.record.class}:"\
-        " #{e.record.errors.full_messages.join(', ')}")
+        user = User.new(user_params)
```

Comment:
I assume this is just so default graphql errors will be used rather than custom ones